### PR TITLE
New check 7165 DynamoDB: DAX encrypted at rest @Daniel-Peladeau @7thseraph @maisenhe

### PR DESCRIPTION
**Pull request by Daniel-Peladeau**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

Check if DynamoDB: DAX Clusters are encrypted at rest. 

Completed with Sean Cai (@7thseraph) and Joel Maisenhelder (@maisenhe)

### CODE REVIEW

**Comment by Daniel-Peladeau** [comment]
Referring to the [ASFF Resource Page](https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-findings-format.html#asff-resources), I was not able to find an ASFF Resource Type for DAX Clusters so the following line was used: `CHECK_ASFF_RESOURCE_TYPE_extra7165="Other"`. If you have a better option to put, I'd be happy to edit it!

File: checks/check_extra7165 [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+#       --iam-role-arn <value>
+#       --sse-specification Enabled=true
+
+
+CHECK_ID_extra7165="7.165"
+CHECK_TITLE_extra7165="[extra7165] Check if DynamoDB: DAX Clusters are encrypted at rest"
+CHECK_SCORED_extra7165="NOT_SCORED"
+CHECK_TYPE_extra7165="EXTRA"
+CHECK_SEVERITY_extra7165="Medium"
+CHECK_ASFF_RESOURCE_TYPE_extra7165="Other"
```
Comment (toniblyx) [inline checks/check_extra7165]: As far as I can see here https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-dax-cluster.html it should be AwsDaxCluster

**Review by toniblyx — CHANGES_REQUESTED** [review]

File: checks/check_extra7165 (hunk 1 above)
Comment (Daniel-Peladeau) [reply checks/check_extra7165]: I was going off that other documentation I linked and missed that. Thank you for the heads up, I have submit the changes.

**Review by toniblyx — APPROVED** [review]

**Comment by toniblyx** [comment]
Thanks guys!

**Pull request merged**